DESCRIPTION

```diff
@@ -20,11 +20,12 @@ Authors@R: c(person("Daniel", "McGlinn", role = c("aut", "cre"),
 BugReports: https://github.com/ropensci/rdataretriever/issues
 URL: https://github.com/ropensci/rdataretriever/
 Depends:
-    R (>= 3.0.0)
+    R (>= 3.3.0)
```

Comment:
Do we have a real requirement for 3.3 or is this Roxygen resetting this? If it's Roxygen let's set this back to 3.0.